integration/spark2/src/main/scala/org/apache/spark/sql/hive/CarbonPreAggregateRules.scala

```diff
@@ -91,6 +91,8 @@ case class CarbonPreAggregateQueryRules(sparkSession: SparkSession) extends Rule
       // create buffer to collect all the column and its metadata information
       val list = scala.collection.mutable.ListBuffer.empty[QueryColumn]
       var isValidPlan = true
+      // REVIEW_COMMENT: should use transform instead of match,
+      // so that it is not just matching the root node in the plan
```

Comment:
We added because here we are not transforming anything this match is to get all the attributes from plan to select the proper aggregate table after that we will transform